index.html

```diff
@@ -1,284 +1,309 @@
-        </div>
-        <div>
-          <p class="about-me">
-            I’m a software developer! I can help you <br />
-            build a product , feature or website Look <br />
-            through some of my work and <br />
-            experience! If you like what you see and <br />
-            have a project you need coded, don’t <br />
-            hestiate to contact me.
-          </p>
-        </div>
-        <div class="contact-me">
-          <h2 class="connect">let's connect</h2>
-        </div>
-        <div>
-          <ul class="buttons">
-          <li><a href=""><img src="./assets/Github.svg"  class="contact-btn" alt="Github" /></li></a>
-          <li><a href=""><img src="./assets/Twitter.svg" class="contact-btn"  alt="Twitter" /></li></a>
-          <li><a href=""><img src="./assets/medium.svg"  class="contact-btn" alt="Medium" /></li></a>
-          <li><a href=""><img src="./assets/Vector.svg"  class="contact-btn" alt="" /></li></a>
+    <section class="my-section">
+      <div>
+        <h1 class="intro">I'm Walid <br />Glad to see You!</h1>
+      </div>
+      <div>
+        <p class="about-me">
+          I’m a software developer! I can help you 
+          build a product , feature or website Look 
+          through some of my work and 
+          experience! If you like what you see and 
+          have a project you need coded, don’t 
+          hestiate to contact me.
+        </p>
+      </div>
+      <div class="contact-me">
+        <h2 class="connect">let's connect</h2>
+      </div>
+      <div>
+        <ul class="buttons">
+          <li><a href=""><img src="./assets/Github.svg" class="contact-btn" alt="Github" /></li></a>
+          <li><a href=""><img src="./assets/Twitter.svg" class="contact-btn" alt="Twitter" /></li></a>
+          <li><a href=""><img src="./assets/medium.svg" class="contact-btn" alt="Medium" /></li></a>
+          <li><a href=""><img src="./assets/Vector.svg" class="contact-btn" alt="" /></li></a>
         </ul>
-        </div>
-      </section>
+      </div>
+    </section>
 
-      <section class="card">
+    <section class="card">
       <div class="grid-container">
         <div class="first-card">
           <img class="card-img" src="/assets/First Snapshoot Portfolio.svg" alt="Tonic">
-          <div class="my-project"> 
-
-          <h2 class="project-name">Tonic</h2>
-          <div class="experience">
-          <span class="tag1">CANOPY</span>
-          <span class="bullet"></span>
-          <span class="tag">Back End Dev</span>
-          <span class="bullet"></span>
-          <span class="tag"> 2015</span>
-          
-        </div>
-       
-        <div class="descreption" >
-          <p> A daily selection of privately
-              personalized reads; no accounts or 
-              sign-up required.</p>
-          </div>
+          <div class="my-project">
+
+            <h2 class="project-name">Tonic</h2>
```

Comment:
- [ ] Let's also fix the font-size for this heading as well, currently it is set to 32px, in the template however it is 40px :point_down: 
![image](https://user-images.githubusercontent.com/97521557/213239201-f66e8ab9-02f8-4726-8fe6-5810366eb167.png)